rails-type-casting.md

```diff
@@ -0,0 +1,62 @@
+Title
+==
+
+I don't know I'm bad at naming things
+
+Abstract
+==
+
+Type casting in Active Record is an integral part of Rails, but very few of us
+know how it works and the changes it has gone through. We're going to unravel
+the mysteries behind how Active Record type casts anything you give to it. We'll
+look at the major changes that happened in Rails 4.2, and show the new API
+coming in Rails 5. We'll be discussing the process of cleaning up complex legacy
+code. Finally, we'll look at the power of simple, composeable APIs, and how this
+affects the future of Rails.
+
```

Comment:
The last sentence hints at why I, as a mere Rails Developer, want to go to this talk. The refactoring is interesting to me, but it's not immediately clear what I'll get out of it.